.gitignore

```diff
@@ -1,3 +1,3 @@
-vendor
-composer.lock
```

Comment:
Our other libraries don't commit `composer.lock` either, so I'd suggest leaving it out for the module as well.